applications/MappingApplication/mapping_application.h

```diff
@@ -139,6 +140,21 @@ class KratosMappingApplication : public KratosApplication
     }
 
 
+
+    static std::unordered_map<std::string, Mapper::Pointer>& GetRegisteredMappersList()
+    {
+        static std::unordered_map<std::string, Mapper::Pointer> registered_mappers;
+
+        return registered_mappers;
+    }
+
+    static void RegisterMapper(const std::string MapperName,
```

Comment:
This should be the `MapperFactory::Register` method and I would put it in the cpp of mapper factory. In future we will move this register to the core to enforce better the singleton and keep track of all objects.